nri/index.html

```diff
@@ -0,0 +1,232 @@
+        </div>
+        <div class="icon-item"><a href="#"><img src="images/icon-twitter.svg" alt="Twitter"></a>
+        </div>
+        <div class="icon-item"><a href="#"><img src="images/icon-linkedin.svg" alt="Linkedin"></a>
+        </div>
+      </div>
+    </article>
+
+    <footer>
+      <div class="corporate_info">
+        <h2>企業情報</h2>
+        <div class="Management policy">
+          <div class="container">
+            <h4>経営方針</h4>
+            <div class="item"><a href="#">社長メッセージ</a></div>
+            <div class="item"><a href="#">企業理念</a></div>
+            <div class="item"><a href="#">経営ビジョン</a></div>
+            <div class="item"><a href="#">品質向上・リスク管理</a></div>
+            <div class="item"><a href="#">人材の育成</a></div>
+            <div class="item"><a href="#">内部統制</a></div>
+            <div class="item"><a href="#">NRIグループ企業行動原則</a></div>
+            <div class="item"><a href="#">NRIグループビジネス行動基準</a></div>
+            <div class="item"><a href="#">コーポレート・ガバナンス</a></div>
+            <div class="item"><a href="#">事業等のリスク</a></div>
+            <div class="item"><a href="#">NRIグループの調達方針</a></div>
+          </div>
+        </div>
+        <div class="Company profile">
+          <div class="container">
+            <h4>企業案内</h4>
+            <div class="item"><a href="#">会社概要</a></div>
+            <div class="item"><a href="#">沿革</a></div>
+            <div class="item"><a href="#">組織図</a></div>
+            <div class="item"><a href="#">役員構成</a></div>
+            <div class="item"><a href="#">未来創発の軌跡</a></div>
+          </div>
+        </div>
+        <div class="strength">
+          <h4>NRIの強み</h4>
+          <div class="item"><a href="#">NRIのあゆみと持続的な成長</a></div>
+          <div class="item"><a href="#">問題発見から問題解決まで</a></div>
+          <div class="item"><a href="#">強固な顧客基盤</a></div>
+          <div class="item"><a href="#">ビジネスモデルの特徴</a></div>
+          <div class="item"><a href="#">NRIのデジタルビジネス戦略</a></div>
+          <div class="item"><a href="#">人材が支えるNRI</a></div>
+          <div class="item"><a href="#">業界トップクラスの収益力</a></div>
+        </div>
+        <div class="group">
+          <h4>NRIグループ国内・海外拠点一覧</h4>
+          <div class="item"><a href="#">メインオフィス</a></div>
+          <div class="item"><a href="#">グループカンパニー</a></div>
+          <div class="item"><a href="#">その他の関連会社・団体</a></div>
+        </div>
+        <div class="Business_info">
+          <h4>事業案内</h4>
+          <div class="item"><a href="#">コンサルティング</a></div>
+          <div class="item"><a href="#">金融ITソリューション</a></div>
+          <div class="item"><a href="#">産業ITソリューション</a></div>
+          <div class="item"><a href="#">IT基盤サービス</a></div>
+        </div>
+      </div>
+      <div class="page-back"><a href="#">前のページに戻る</a></div>
+      <div class="follow_us">
+        <h2>Follow Us</h2>
+        <div class="icon-item"> <a href="#"><img src="images/yt_icon_rgb.svg" alt="YouTube">
```

Comment:
余計なスペースがあるので削除してください。

```suggestion
        <div class="icon-item"><a href="#"><img src="images/yt_icon_rgb.svg" alt="YouTube"></a></div>
```